client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -84,8 +128,25 @@ class RequiredPluginsInstallView extends Component {
 		};
 	};
 
+	doTransferStatusPolling = () => {
+		const { automatedTransferStatus, siteId } = this.props;
+
+		const { COMPLETE } = transferStates;
+
+		if ( automatedTransferStatus === COMPLETE ) {
```

Comment:
Maybe yoda

Reply:
In JS? :laughing:

Reply:
Its optional, but we tend to do it in woocommerce calypso code

Reply:
Updated in de62ee5f89.

> Its optional, but we tend to do it in woocommerce calypso code

Interestingly enough, there are some places in the file where yoda is not used